src/docs/frontend/index.mdx

```diff
@@ -128,8 +128,6 @@ const SomeComponent = styled('div')`
 export default SomeComponent;
 ```
 
-* Note that `reflexbox` (e.g. `Flex` and `Box`) is being deprecated, avoid using in new code.
```

Comment:
Pulled this out because `reflexbox` seems to be gone as of https://github.com/getsentry/sentry/commit/5d919dc73b06529230ede630afda90f81eae78d6